composer.json

```diff
@@ -26,5 +28,27 @@
         "psr-4": {
             "RebelCode\\Modular\\": "src"
         }
-    }
+    },
+    "scripts": {
+        "test": "phpunit",
```

Comment:
PHPUnit and CS Fixer are declared as dev dependencies everywhere, and therefore live in the _vendor/bin_ folder. This is the reliable way to invoke them.